pkg/controller/controller.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/operator-framework/combo/api/v1alpha1"
 )
 
-const Version = 1
+const verbosity = 1
```

Comment:
I think this needs to be a CLI flag that users can configure and not an exported constant variable that's shared throughout the controller package.

Reply:
Yeah, good call.

Reply:
After working on this for a bit it seems this would be best addressed in another PR. Logging and its levels across the repo need to be checked. I went ahead and made this an issue so we can track that work.

https://github.com/operator-framework/combo/issues/62